seleniumDescriptors.js

```diff
@@ -0,0 +1,177 @@
+var format = require('util').format;
```

Comment:
It clutters up the repository, making the organization more difficult to understand. Also, the descriptor code doesn't have a different responsibility. It's sole purpose is simply to provide configuration information to the SeleniumTunnel class. It (currently) doesn't even need to be exported by SeleniumTunnel since it isn't meant to be used directly by testers.